apps/web/src/lang/fr.json

```
@@ -5282,6 +5306,10 @@
     "defaultMessage": "Créer un ministère",
     "description": "Title displayed on the create a department form."
   },
+  "XBfpuW": {
+    "defaultMessage": "Ouvrir",
```

I am pretty sure the English "Open" refers to the adjective and so _ouvert_ (the adjective) should be use and not _ouvert_ (the verb).
```suggestion
    "defaultMessage": "Ouvert",
```